api/envoy/api/v2/route/route.proto

```diff
@@ -1076,6 +1076,20 @@ message Tracing {
   // :ref:`HTTP Connection Manager <config_http_conn_man_runtime>`.
   // Default: 100%
   envoy.type.FractionalPercent overall_sampling = 3;
+
+  message CustomTag {
```

Comment:
To reviewers, i have no idea where to place it for a common use, so duplicated here.
Do you have any thoughts? Thanks.

Reply:
Only thought would be a shared proto definition somewhere that is imported to both locations. But best if one of the @envoyproxy/api-shepherds comment.

Reply:
I prefer `api/envoy/type` also, just not sure whether the `CustomTag` qualifies it.
Waiting for a sure comment, :).

Reply:
moved to `api/envoy/type`